src/library/Exceptions.nim

```diff
@@ -0,0 +1,148 @@
+
+{.used.}
+
+import vm/lib
+import vm/exec
+import vm/errors
+import vm/values/custom/verror
+
+proc defineSymbols*() =
+
+    constant "genericError",
```

Comment:
Re: this.

Just noticed it too. I love it that you included an error as a constant (to get it started). Now, why do I keep thinking that I might like something like `GenericError` for this type of thing? (I'm not telling you to do - or not do anything - I'm just thinking out loud, kind of :) )